ci_scripts/ci/circle/run.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# FIXME: transition
+API_BASE="https://api.github.com/repos/scottwittenburg/sensei"
+COMMIT=${CIRCLE_SHA1}
+CDASH_STATUS_CONTEXT="cdash"
+PYTHON_EXECUTABLE=python
+
+build_status_body() {
+  cat <<EOF
+{
+  "state": "success",
+  "target_url": "https://cdash.nersc.gov/index.php?compare1=61&filtercount=1&field1=revision&project=sensei&showfilters=0&limit=100&value1=${COMMIT}&showfeed=0",
+  "description": "Build and test results available on CDash",
+  "context": "${CDASH_STATUS_CONTEXT}"
+}
+EOF
```

Comment:
I think these "here doc" implement the badge feature that you mentioned are in a future release of cdash. would you please add some comments explaining what these functions do and in which versions they become obsolete and if practical the steps to replace with the official feature?

Reply:
I'll add some explanation of what this is doing and how it can be replaced when next there is a CDash release and NERSC upgrades to use it.  Though to be clear, this isn't about badges, but just about posting the link to the CDash build results in the PR status checks.